Changelog.md

```diff
@@ -1,7 +1,7 @@
 ### 0.8.3 (unreleased)
 
 Language Features:
-
+ * Possibility to use ``bytes.concat`` with variable number of ``bytes`` and ``FixedBytes`` arguments.
```

Comment:
You should describe what it does.

Reply:
Maybe just mention that it is a restricted version of ``abi.encodePacked`` with a more descriptive name?

Reply:
I mentioned like that, hope it looks good enough now